Tools/setup.py

```diff
@@ -48,7 +48,7 @@
     # 'otps',  # Hard to install, but available on Sandbox and NCI
     # datahandling
     'GDAL',
-    'odc',
+    'odc-tools',
```

Comment:
Can you do `pip list | grep 'odc'` and check what exactly was installed on the sandbox or NCI? I did mine.
```
jovyan@jupyter-anotheremma:~$ pip list | grep 'odc'
odc-aio                           0.1.dev674+g2936179
odc-algo                          0.1.dev674+g2936179
odc-apps-cloud                    0.1.dev674+g2936179
odc-apps-dc-tools                 0.1.dev674+g2936179
odc-aws                           0.1.dev674+g2936179
odc-dscache                       0.1.dev674+g2936179
odc-dtools                        0.1.dev674+g2936179
odc-geom                          0.1.dev674+g2936179
odc-index                         0.1.dev674+g2936179
odc-io                            0.1.dev674+g2936179
odc-ppt                           0.1.dev674+g2936179
odc-stats                         0.1.dev674+g2936179
odc-thredds                       0.1.dev674+g2936179
odc-ui                            0.1.dev674+g2936179
```

Reply:
oh they're all separate? okey, can do